### PR TITLE
Add links to official docs across compute Terraform Google Provider docs

### PR DESCRIPTION
**Pull request by rosbo**
cc @rileykarson @danawillow

### CODE REVIEW

File: website/docs/r/compute_project_metadata.html.markdown [hunk 1]
```diff
@@ -6,7 +6,10 @@ description: |-
   Manages common instance metadata
 ---
 
-# google\_compute\_project\_metadata
+# google\_compute\_project\_metadata. For more information see
```
Comment (rileykarson) [inline website/docs/r/compute_project_metadata.html.markdown]: Looks like this got applied to the wrong line.

File: website/docs/r/compute_route.html.markdown [hunk 2]
```diff
@@ -8,7 +8,10 @@ description: |-
 
 # google\_compute\_route
 
-Manages a network route within GCE.
+Manages a network route within GCE. For more information see
+[the official documentation](https://cloud.google.com/compute/docs/vpc/routes)
+and
+[API](https://cloud.google.com/compute/docs/reference/latest/projects/routes).
```
Comment (rileykarson) [inline website/docs/r/compute_route.html.markdown]: I get a 404 here - did you mean https://cloud.google.com/compute/docs/reference/latest/routes?

File: website/docs/r/compute_router.html.markdown [hunk 3]
```diff
@@ -8,8 +8,10 @@ description: |-
 
 # google\_compute\_router
 
-Manages a Cloud Router resource. For more info, read the
-[documentation](https://cloud.google.com/compute/docs/cloudrouter).
+Manages a Cloud Router resource. For more information see
+[the official documentation](https://cloud.google.com/compute/docs/cloudrouter)
+and
+[API](https://cloud.google.com/compute/docs/reference/latest/projects/routers).
```
Comment (rileykarson) [inline website/docs/r/compute_router.html.markdown]: I get a 404 here - did you mean https://cloud.google.com/compute/docs/reference/latest/routers?

File: website/docs/r/compute_router_interface.html.markdown [hunk 4]
```diff
@@ -8,8 +8,10 @@ description: |-
 
 # google\_compute\_router_interface
 
-Manages a Cloud Router interface. For more info, read the
-[documentation](https://cloud.google.com/compute/docs/cloudrouter).
+Manages a Cloud Router interface. For more information see
+[the official documentation](https://cloud.google.com/compute/docs/cloudrouter)
+and
+[API](https://cloud.google.com/compute/docs/reference/latest/projects/routers).
```
Comment (rileykarson) [inline website/docs/r/compute_router_interface.html.markdown]: Same as above

File: website/docs/r/compute_router_peer.html.markdown [hunk 5]
```diff
@@ -8,8 +8,10 @@ description: |-
 
 # google\_compute\_router
 
-Manages a Cloud Router BGP peer. For more info, read the
-[documentation](https://cloud.google.com/compute/docs/cloudrouter).
+Manages a Cloud Router BGP peer. For more information see
+[the official documentation](https://cloud.google.com/compute/docs/cloudrouter)
+and
+[API](https://cloud.google.com/compute/docs/reference/latest/projects/routers).
```
Comment (rileykarson) [inline website/docs/r/compute_router_peer.html.markdown]: Same as above

File: website/docs/r/compute_snapshot.html.markdown [hunk 6]
```diff
@@ -8,7 +8,10 @@ description: |-
 
 # google\_compute\_snapshot
 
-Creates a new snapshot of a disk within GCE.
+Creates a new snapshot of a disk within GCE. For more information see
+[the official documentation](https://cloud.google.com/compute/docs/disks/create-snapshots)
+and
+[API](https://cloud.google.com/compute/docs/reference/latest/projects/snapshots).
```
Comment (rileykarson) [inline website/docs/r/compute_snapshot.html.markdown]: Did you mean https://cloud.google.com/compute/docs/reference/latest/snapshots?

File: website/docs/r/compute_router_interface.html.markdown [hunk 7]
```diff
@@ -8,8 +8,10 @@ description: |-
 
 # google\_compute\_router_interface
 
-Manages a Cloud Router interface. For more info, read the
-[documentation](https://cloud.google.com/compute/docs/cloudrouter).
+Manages a Cloud Router interface. For more information see
+[the official documentation](https://cloud.google.com/compute/docs/cloudrouter)
```
Comment (rileykarson) [inline website/docs/r/compute_router_interface.html.markdown]: Not needed for LGTM but nice-to-have: is there a header we can link to in the docs that is more relevant than the Cloud Router page as a whole?

File: website/docs/r/compute_project_metadata.html.markdown (hunk 1 above)
Comment (rosbo) [inline website/docs/r/compute_project_metadata.html.markdown]: Done

File: website/docs/r/compute_route.html.markdown (hunk 2 above)
Comment (rosbo) [inline website/docs/r/compute_route.html.markdown]: Yup. Fixed

File: website/docs/r/compute_router.html.markdown (hunk 3 above)
Comment (rosbo) [inline website/docs/r/compute_router.html.markdown]: Fixed

File: website/docs/r/compute_router_interface.html.markdown (hunk 4 above)
Comment (rosbo) [inline website/docs/r/compute_router_interface.html.markdown]: Fixed

File: website/docs/r/compute_router_peer.html.markdown (hunk 5 above)
Comment (rosbo) [inline website/docs/r/compute_router_peer.html.markdown]: Fixed

File: website/docs/r/compute_snapshot.html.markdown (hunk 6 above)
Comment (rosbo) [inline website/docs/r/compute_snapshot.html.markdown]: Yes. Duh.

File: website/docs/r/compute_router_interface.html.markdown (hunk 7 above)
Comment (rosbo) [inline website/docs/r/compute_router_interface.html.markdown]: Not really...

Comment (rileykarson) [inline website/docs/r/compute_router_interface.html.markdown]: SG :+1:

**Pull request merged**